### DISCUSSION TITLE
Row Level Security

### DISCUSSION THREAD

**Discussion by ghost** [supabase/supabase#5844] [Questions]
new row violates row-level security policy for table "objects"

in next js, it coming error

**Comment by GaryAustin1** [comment]
That likely means you have not set your policies (RLS) for your storage.
https://supabase.com/docs/guides/storage#allow-public-access-to-a-bucket
If you are trying to upload a file you will need an insert policy. https://supabase.com/docs/reference/javascript/storage-from-upload#notes

**Reply by ghost** [reply]
Thanks
